clients/redis/command.go

```diff
@@ -0,0 +1,308 @@
+//
+// Copyright (C) 2022 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package redis
```

Comment:
This client works with any of the MessageBus implementation, so package should just be clients.